pyproject.toml

```
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 100
```

```suggestion
[tool.black]
line-length = 100
```
I don't think we're going down the pyproject.toml route. But we're open to be persuaded that we should!

---

I commited this file trying to solve the issue with black that we discussed earlier with thomke, I'm not sure if this is the path to go either, I'll remove the file from this PR as it can be discussed later